web-wallet/src/lib/stores/gasStore.js

```diff
@@ -0,0 +1,11 @@
+import { readable } from "svelte/store";
+
+const gasSettings = {
+	limitLower: Number(parseInt(import.meta.env.VITE_GAS_LIMIT_LOWER, 10)),
```

Comment:
Why are you converting to number something that is already a number?

If you wanted to handle the `NaN` case, this is wrong as `Number(NaN)` will give you back a `NaN`.

I would go for something like:

```js
const envLimitLower = parseInt(import.meta.env.VITE_GAS_LIMIT_LOWER, 10);

// ...

const gasSettings = {
  limitLower: isNaN(envLimitLower) ? /* some default value */ : envLimitLower,
  
  // ...
};
```

If instead we just assume that we have the correct env variables you can simply remove the number conversion.

Reply:
I think this something that I left in by accident when I was testing some things out. `Number()` is not needed at all here.